### PR TITLE
[ISSUE #4084]Method prints the stack trace to the console[EventMeshMessageProducer]

### LINKED ISSUE #4084: [Enhancement] Method prints the stack trace to the console[EventMeshMessageProducer]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/eventmesh/issues?q=is%3Aissue) and found no similar issues.

### Enhancement Request

image
located at:
eventmesh-sdks/eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/grpc/producer/EventMeshMessageProducer.java line 115
analysis and explanation:
This method prints a stack trace to the console. This is non configurable, and causes an application to look unprofessional. Switch to using loggers so that users can control what is logged and where.

### Describe the solution you'd like

delete the stack print.

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### PR DESCRIPTION
**Pull request by harshithasudhakar**
Fixes #4084 

### Modifications

Removed stack trace at line 119

### Documentation

- Does this pull request introduce a new feature? (no)
- If yes, how is the feature documented? (not applicable)

### CODE REVIEW

File: eventmesh-sdks/eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/grpc/producer/EventMeshMessageProducer.java [hunk 1]
```diff
@@ -116,7 +116,6 @@ public EventMeshMessage requestReply(EventMeshMessage message, long timeout) {
             }
             return EventMeshCloudEventBuilder.buildMessageFromEventMeshCloudEvent(reply, PROTOCOL_TYPE);
         } catch (Exception e) {
-            e.printStackTrace();
             if (log.isErrorEnabled()) {
                 log.error("Error in RequestReply message {}", message, e);
             }
```
Comment (mxsm) [inline eventmesh-sdks/eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/grpc/producer/EventMeshMessageProducer.java:121]: Consider removing the if statement.

**Review by mxsm — CHANGES_REQUESTED** [review]

**Review by mxsm — APPROVED** [review]
LGTM

**Comment by mxsm** [comment]
@harshithasudhakar CI not pass! please check [ant:checkstyle] [WARN] /Users/runner/work/eventmesh/eventmesh/eventmesh-sdks/eventmesh-sdk-java/src/main/java/org/apache/eventmesh/client/grpc/producer/EventMeshMessageProducer.java:119:17: 'catch' child has incorrect indentation level 16, expected level should be 12. [Indentation]

**Review by Pil0tXia — APPROVED** [review]
LGTM

**Review by mxsm — APPROVED** [review]

**Review by Alonexc — APPROVED** [review]

**Review by xwm1992 — APPROVED** [review]

**Review by lrhkobe — APPROVED** [review]

**Pull request merged**